invenio_stats/aggregations.py

```diff
@@ -332,8 +349,10 @@ def agg_iter(self, dt):
             terms.metric("top_hit", "top_hits", size=1, sort={"timestamp": "desc"})
             for dst, (metric, src, opts) in self.metric_fields.items():
                 terms.metric(dst, metric, field=src, **opts)
+            # Let's get also the last time that the event happened
```

Comment:
This one is the important part of the PR! It makes sure that the entries that have already been calculated are not aggregated again.